spec.html

```diff
@@ -25358,10 +25361,10 @@ <h1>Number.isSafeInteger ( _number_ )</h1>
 
       <emu-clause id="sec-number.max_safe_integer">
         <h1>Number.MAX_SAFE_INTEGER</h1>
+        <p>The value of Number.MAX_SAFE_INTEGER is 9007199254740991 (2<sup>53</sup> - 1).</p>
         <emu-note>
-          <p>The value of `Number.MAX_SAFE_INTEGER` is the largest integer n such that n and n + 1 are both exactly representable as a Number value.</p>
+          <p>Integers greater than `Number.MAX_SAFE_INTEGER` cannot by uniquely represented by Number values and are therefore not necessarily distinguishable. For example, both `9007199254740992` and `9007199254740993` identify the Number value 9007199254740992.</p>
```

Comment:
I like the added clarity in this PR, but I'm not sure what "identify" means in this context. Would something like "are represented by" be more precise?

Reply:
I thought about that too, but it read weird to me when the same sentence referenced integers being represented by Number values and Number values being represented by ECMAScript literals. I'm open to other ideas, though, and can change it if you feel strongly.

Reply:
Perhaps say they are equal/identical values?

Reply:
I changed "identify" to "evaluate to".